app/src/test/java/com/readtracker/android/db/export/ExportedFileParserTest.java

```diff
@@ -1,39 +1,48 @@
 package com.readtracker.android.db.export;
 
+import android.test.AndroidTestCase;
+
 import com.readtracker.android.db.Book;
+import com.readtracker.android.test_support.SharedExampleAsserts;
 import com.readtracker.android.test_support.TestUtils;
 
-import junit.framework.TestCase;
-
 import org.json.JSONException;
+import org.junit.Ignore;
+import org.junit.Test;
 
 import java.util.List;
 
-public class ExportedFileParserTest extends TestCase {
+public class ExportedFileParserTest extends AndroidTestCase {
 
-  public void test_build_book_from_json() throws Exception {
+  @Test
+  public void exportedFileParserTest_buildBookFromJson_returnListOfBooks() throws Exception {
     List<Book> books = _parseTestFile("export_version_2.json");
     SharedExampleAsserts.assertExampleBooksVersion2(books);
   }
 
+  @Test
   /**
    * Check that the parser handles null in json correctly.
    */
-  public void test_build_null_values_from_json() throws Exception {
+  public void exportedFileParserTest_buildBookFromJsonWithNullCases_returnListOfBooks() throws Exception {
     List<Book> books = _parseTestFile("export_version_2_null_fields.json");
     assertEquals(1, books.size());
     Book book = books.get(0);
     SharedExampleAsserts.assertNullBookVersion2(book);
   }
 
-  /** Check that the parser handles missing fields json correctly. */
-  public void test_build_missing_values_from_json() throws Exception {
+  @Test
+  /**
+   * Check that the parser handles missing fields json correctly.
+   */
+  public void exportedFileParserTest_buildBookFromJsonWithMissingValues_returnListOfBooks() throws Exception {
     List<Book> books = _parseTestFile("export_version_2_missing_fields.json");
     assertEquals(1, books.size());
     Book book = books.get(0);
     SharedExampleAsserts.assertNullBookVersion2(book);
   }
 
+  @Ignore
```

Comment:
The only place we really need it is the top of the DatabaseTestCase class, otherwise JUnit will try to run it due to the Setup and Teardown methods being present there. Will remove the rest.